test/GhSyncTests/AdoTests.cs

```diff
@@ -1,18 +1,49 @@
+namespace gh_sync.Tests;
+
 using Xunit;
 using gh_sync;
-using Octokit;
 using System;
+using Octokit;
 using System.Threading.Tasks;
+using System.Linq;
 using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using Microsoft.Extensions.DependencyInjection;
 
-
-public class AdoTests
+public record class AdoTests(MockStartup Startup) : IClassFixture<MockStartup>
 {
+    private IAdo MockAdo => Startup.Services.GetRequiredService<IAdo>();
+    Ado Ado = new Ado();
```

Comment:
Since this looks to test the live `Ado` object, my recommendation would be to make sure to create that object from the services object, and to remove the MockAdo.

```suggestion
    private readonly Ado Ado = ActivatorUtils.CreateInstance<Ado>(Startup.Services);
```